articles/step-by-step-on-how-to-user-nodejs-repl-tutorial/index.md

```diff
@@ -0,0 +1,203 @@
+
+ 1. Performing Arithmetical operations in REPL
+In the previous example, we printed 'hello world' string on the screen, in this section let's dive in and look at arithmetics:  
+
+In your terminal, start REPL: 
+
+```bash
+$ node
+> 
+
+```
+Let's perform basic addition, subtraction,modulus, division and multipliation respectively:
+Output:  
+
+```bash
+> 2+2
+4
+> 5-2
+3
+> 10%3
+1
+> 20/2
+10
+> 10*2
+20
+> 
+
+```
+2. Performing operations using Node's Math library
+Math library is an object with defined methods and properties. We can therefore use these features to perform some arithmetics.  
+
+Example:
+
+```bash
+---skipped lines
+
+> function getRandomNumber(maximum){
+... return Math.floor(Math.random()*maximum);
+... }
+undefined
+
+```
+
+Output:
+```bash
+> console.log(getRandomNumber(20));
+12
+undefined
+> 
+```
+In REPL, we define `getRandomNumber(args)` and pass it a maximum number. The method uses `Math.floor()` to return a random largest integer less than or equal to a given number.
+
+> It's important to note that `Math.random()` does not provide secure random numbers. Do not use them on security-related tasks.
+
+
+3. Calling Methods (Functions) Using REPL
+
+Normally, we write functions to handle specific tasks. REPL provides an easy way to handle these methods.   
+In JavaScript, we commonly use the `console.log()` global method to print messages. 
+
+Let's enter the following in the prompt: 
+
+```bash
+$ node
+> console.log('Hello world');
+```
+Output: 
+
+```bash 
+Hello world
+undefined // this is the return value of this method, console.log()
+> 
+```
+
+When you press ENTER, the output is displayed as shown above.  
+
+Next, let's create a function to add two numbers:  
+
+```bash
+--- skipped lines
+
+> function addTwoNumbers(firstNumber,lastNumber){
+... console.log(firstNumber+ lastNumber)
+... }
+undefined
+
+```
+We've a function `addTwoNumbers(arguments)` which takes two arguments, `firstNumber` and `lastNumber`.
+This function, whenever it's called, logs out the addition of these two numbers.  
+
+Output:
+
+```bash
+---- skipped lines
+> addTwoNumbers(20,40);
+60
+undefined
+> 
+```
+
+We call this method and pass it two parameters, `addTwoNumbers(20,40)`( you're free to pass any integer value).
+By pressing ENTER, the result is displayed on the screen as shown above.  
+
+
+4. Creating Variables using REPL
+
+Creating variables works the same as it would in your`.js` file.
+Let's look at an example:  
+
+```bash
+$node
+> let name = 'john doe';
+```
+Output on calling `name` and pressing ENTER:
+
+```bash
+undefined
+> name
+'john doe'
+> 
+
+```
+This variable `name` will remain active until you exit the REPL window session. This implies that you can even concatenate this string to another.  
+For instance:
+
+```bash
+$node
+> let name = 'john doe';
+undefined
+> name
+'john doe'
+> name+ ' is a student at Stanford';
+'john doe is a student at Stanford'
+> 
+
+
+```
```

Comment:
Almost all the code snippets looks unclear, since there's frequent use of `$` and `>`.

I would suggest to put screenshots of the code snippets instead, which would enhance the readability of the code snippets.